Scripts/edit_user_info.php

```diff
@@ -12,32 +12,59 @@
 $emailUpdated = false;
 
 if (strlen($firstName) > 0) {
+    // Get old first name for records
+    $logsql = "SELECT `FirstName` FROM `PortfolioTracker`.`Users` WHERE `Users`.`ID`='$userID';";
+    $oldFirstName = $conn->query($logsql); // Gives old first name
+    
     $sql = "UPDATE `PortfolioTracker`.`Users` SET `FirstName`='$firstName' WHERE `Users`.`ID`='$userID'";
     $result = $conn->query($sql); // Returns result of query
     if ($result) {
         $firstNameUpdated = true;
+        
+        // Only update Logs if query successful
+        $log = "INSERT INTO `PortfolioTracker`.`Logs` (`ID`, `Change`, `Date`, `OldInfo`, `NewInfo`) VALUES
+                ('$userID', 'FirstNameChange', NOW(), '$firstName', '$oldFirstName');";
```

Comment:
Same here as in create_script.php. Call fetch_row() on $oldFirstName and index it when adding it db in query.